assets/blocks/Underglaze.block

```diff
@@ -4,11 +4,11 @@
     "categories" : ["Brick"],
     "template" : false,
     "shape" : "engine:cube",
-    "hardness": 50    
+    "hardness": 50,    
     "attachmentAllowed" : true,
     "supportRequired" : false,
     "penetrable" : false,
     "targetable" : true,
     "invisible" : false,
     "mass" : 40
-}
\ No newline at end of file
+}

```

Comment:
Ah, yeah, I ended up changing the file through the GitHub web interface because it was such a simple fix once I knew where the issue was. Apparently the web UI adds a windows newline o_o